stable/distributed-tensorflow/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart for running distributed TensorFlow on Kubernetes
 name: distributed-tensorflow
-version: 0.1.3
-appVersion: 1.6.0
+version: 0.1.4
+appVersion: 1.7.0
 sources:
 - https://github.com/tensorflow/tensorflow
 - https://www.tensorflow.org/deploy/distributed
```

Comment:
I also suggest to change 'home' to this github repo, and not pointing to tensorflow website - it makes it easier to find the owners of the chart in chart hub.

Reply:
Done